app/views/landlords/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="container">
```

Comment:
checkout out `views/devise/registrations/new.html.erb` This is probably were we will have the user sign up stuff and conditionally display the landlord fields or the tenant fields depending on what kind of user they want to be.